src/modules/controllers/task.controllers.js

```diff
@@ -0,0 +1,24 @@
+const Expenses = require("../../schema/taskSchema");
+
+module.exports.getAllExpenses = (req, res) => {
+  try {
+    Expenses.find().then((result) => {
+      res.send({ data: result });
+    });
+  } catch (e) {
+    res.status(500).send({ message: "Internal server error" });
+  }
+};
+
+module.exports.createNewExpense = (req, res) => {
+  try {
```

Comment:
if i don't send body? or if i send empty body? where do you take into account the creation date of the record?